internal/storage/gcs/bucket.go

```diff
@@ -20,6 +20,15 @@ import (
 	"golang.org/x/net/context"
 )
 
+// BucketTypes represents different types of buckets.
```

Comment:
should be changed to "BucketType represent type of bucket like Hierarchical nonHierarchical as constants"